expo/scheduler/functions.py

```diff
@@ -435,15 +411,22 @@ def date_to_datetime(the_date):
     return utc.localize(datetime.combine(the_date, zero_hour))
 
 
+def send_today():
```

Comment:
Because it built off today's date, I got stuck on the idea being about today.  But you're right...

I did it in the throes of fixing some pesky bug, and missed a good name... here's another try, if you have a better name, let me know..